### DISCUSSION TITLE
Does the lib support stream SRT?

### DISCUSSION THREAD

**Discussion by krisadamqdc** [pedroSG94/RootEncoder#979] [General]
Does the lib support stream SRT?

**Comment by pedroSG94** [comment]
No,

I haven't a good free server to test it (this is the main reason) and compile/implement SRT for Android seems difficult.
It could be supported in the future but for now I want to develop others features because SRT is a really good protocol but is is not supported in most of servers.
Currently, my priority is a mode that allow change video and audio source on fly and video source rotation using sensors. Also, I want develop iOS version of this project (it is a bit abandoned for time reasons) so use this time on SRT is not worth time for now.
